polkadot/node/core/runtime-api/src/lib.rs

```diff
@@ -652,5 +656,13 @@ where
 			ver = Request::CLAIM_QUEUE_RUNTIME_REQUIREMENT,
 			sender
 		),
+		Request::Constraints(para, sender) => {
```

Comment:
I guess this would be affected by this: https://github.com/paritytech/polkadot-sdk/issues/64 at the block the runtime is enacted ? 

So we need to make sure the node using it is robust with garbage data for a block ?

Reply:
There is no storage migration involved so it should not be affected.